apps/admin-panel/src/index.css

```
@@ -4,6 +4,122 @@
+
+.panel-title {
+  font-size: 28px;
+  font-weight: 700;
+  margin: 0;
+}
+
+.panel-desc {
+  margin-top: 8px;
+  font-size: 16px;
+  line-height: 1.6;
+  line-height: 26px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

line-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)